### DISCUSSION TITLE
How to use 'scope' property in SchemaField?

### DISCUSSION THREAD

**Discussion by RihuaChen** [alibaba/formily#1402] [Q&A 2.x] [answered]
I want to only consumer schema of the child component , I found the property scope in SchemaField, it is an any type, I don't know how to use it, pls kindly help me with this situation

![image](https://user-images.githubusercontent.com/11918443/117981571-b84d2900-b367-11eb-8370-67b8566abe9c.png)

**Comment by janryWang** [comment, marked as answer]
```
<SchemaField scope={{locale:{AA:"This is aa"}}} schema={{
  type:"object",
  properties:{
   aa:{
    type:"string",
    title:"{{locale.AA}}"
   }
  }
}}
```

**Reply by RihuaChen** [reply]
Got it, maybe I make the misunderstanding with the scope... 

Just refer to the below schema, my childCmp is rendered by a full schema. Because I just add a real formily page into a formily component.

My real question is I want to submit the values from child cmp but I don't want to get the path 'parentWrapper' in my ChildCmp.

That mean's I want to get 'xxx' but not 'parentWrapper.xxx', is there a way to implement this situation...

```
{
  type: 'object',
  properties: {
    parentWrapper: {
      type: 'object',
      properties: {
        childCmp: {
          'x-component': 'ChildCmp',
          'x-schema': {
            type: 'object',
            properties: {
              xxx,
            },
          },
        },
      },
    },
  },
}
```

**Reply by janryWang** [reply]
set parentWapper.type = void
